machine_management/managing-infrastructure-machinesets.adoc

```diff
@@ -0,0 +1,66 @@
+:context: managing-infrastructure-machinesets
+
+toc::[]
+
+Infrastructure machine sets provide a way to create and manage infrastructure nodes, which allow you to isolate infrastructure workloads. Managing your infrastructure workloads on infra nodes has several benefits.
+
+include::modules/infrastructure-components.adoc[leveloffset=+1]
+
+== Workload placement for infra nodes
+
+The two primary reasons for infra nodes include:
+
+* Preventing incurring billing costs against subscription counts.
+* Separating maintenance workloads and management workloads.
+
+After creating an infrastructure machine set, the new hosted infrastructure nodes have both the `worker` and `infra` roles applied to them. The `worker` role is required when the node is created, and is also used during node upgrades. Nodes with the `infra` role applied are not counted toward the total number of subscriptions that are required to run the environment, even when the `worker` role is also applied. The `worker` role can be removed, but it requires additional procedures to reconfigure the associated machine config pool to manage node upgrades.
+
+[IMPORTANT]
+====
+Unlike earlier versions of {product-title}, you cannot move the infrastructure components to the control plane machines. To move the components, you must create a new machine set.
+====
+
+With an infra node being assigned as a worker, there is a chance user workloads could get inadvertently assigned to an infra node. To prevent other workloads from being scheduled onto infrastructure nodes, there are two possible workflows:
+
+* Apply a taint to the infrastructure nodes and tolerations to the desired infrastructure workloads.
+* Apply a completely separate label to your other nodes and matching node selector to your other workloads such that they are mutually exclusive from infrastructure nodes.
+
+This choice mostly depends on the infrastructure components you plan to assign to the infra nodes. Several infrastructure components do not support configuring taints and tolerations; in those cases, you must apply a separate label to your other nodes so they are mutually exclusive from infrastructure nodes.
+
+include::modules/node-role-management.adoc[leveloffset=+1]
+
+[id="assigning-machineset-resources-to-infra-nodes"]
+== Ensuring resources are scheduled to infrastructure nodes
+
+You can ensure only infrastructure workloads are scheduled to infra nodes by applying a taint to the infrastructure nodes and tolerations to the desired infrastructure workloads. If you prefer to avoid taints, this is also possible through a specified scheduling behavior. You can schedule your own workloads to run on specific non-infrastructure nodes by applying a distinct label to the other nodes and either updating your default scheduler to choose this particular node label or specifically request this node label by project namespace.
+
+include::modules/binding-infra-node-workloads-using-taints-tolerations.adoc[leveloffset=+2]
```

Comment:
Depending on what in in this module, we may want to elaborate more in this section on what to do, explicitly. 

As you say above this doesn't work for all 'components' and as such using this method should be considered carefully (secondary to using a label). 

We don't seem to document this (as far as I can tell), so have we focused on the wrong 'suggestion'?